### DISCUSSION TITLE
What's the best way to update K2ForJ4 to the latest version?

### DISCUSSION THREAD

**Discussion by GixxerMKD** [eworkers/K2ForJ4#75] [Q&A]
Hi guys, i'm looking forward the pull request for the admin style (https://github.com/eworkers/K2ForJ4/pull/66), and started wondering what's the best way to update the extension once that's added? For test i tried to download the zip file (Code/Download Zip) and simply install it again but i ended up restoring backup on the staging site since it broke the site completely.

**Comment by eworkers** [comment]
At the moment the best way to update is to download code as zip, then use the "Upload Package File" in Joomla to install.

Not sure what you did to break a running website using K2ForJ4; but K2ForJ4 cannot do that, it must be something else.
